applications/DEMApplication/custom_utilities/create_and_destroy.cpp

```diff
@@ -1338,6 +1338,46 @@ SphericParticle* ParticleCreatorDestructor::SphereCreatorForBreakableClusters(Mo
         KRATOS_CATCH("")
     }
 
+    void ParticleCreatorDestructor::MarkSandProductionParticlesForErasing(ModelPart& r_model_part) {
```

Comment:
What is inside this function is so problem specific that we should find somewhere else to put it. Any proposal?
Maybe a file called custom_utilities/sand_production_utilities.cpp ?

Reply:
It seems also right to me